src/parser/parser_visitor.py

```diff
@@ -349,6 +353,9 @@ def visitQuerySpecification(
                     from_clause = clause.get('from', None)
                     where_clause = clause.get('where', None)
 
+                elif rule_idx == evaql_parser.RULE_orderByClause:
+                    orderby_clause = self.visit(child)
```

Comment:
Lets replace this with`self.visit(ctx.orderByClause())`. We can also change the other visit calls. The existing code was written when we didn't have a good understanding of the parser. I feel explicitly specifying the child we are visiting improves the code readability.